layouts/partials/site-header.html

```diff
@@ -119,7 +119,7 @@ <h3 class="font-bold text-3xl lg:text-5xl">{{ .Title }}</h3>
                 {{ end }}
             </div>
             <div class="md:mx-auto">
-                <img src="img/white-logo.png" alt="Rotational Labs logo" class="sm:mt-24 h-12 sm:h-40 md:h-64 w-auto">
+                <img src="img/vertical-logo.png" alt="Rotational Labs logo" class="sm:mt-[4.5rem]-12 sm:h-40 md:h-64 w-auto">
```

Comment:
I separated the logo from the text as it would've felt redundant to have the name appear a second time in the header.